velox/exec/tests/HashJoinTest.cpp

```diff
@@ -7065,7 +7065,7 @@ DEBUG_ONLY_TEST_F(HashJoinTest, taskWaitTimeout) {
   const auto expectedResult =
       runHashJoinTask(vectors, nullptr, false, numDrivers, pool(), false).data;
 
-  for (uint64_t timeoutMs : {1'000, 30'000}) {
+  for (uint64_t timeoutMs : {1'000, 60'000}) {
```

Comment:
I don't know why but 30 secs isn't enough with this change on CI machine (`8-core-ubuntu-22.04`). (This isn't needed on my local machine.)

`v2025.09.15.00` FB OS is slower than `v2025.04.28.00`?

I think that FBThrift isn't related because hash join doesn't use Parquet.